README.rst

```diff
@@ -159,7 +159,9 @@ Exclude from search and elevation
 ---------------------------------
 
 By default this add-on introduces two new fields to the default content types
-or any custom type derived from ATContentTypes.
+or any custom type derived from ATContentTypes. For dexterity content types,
+including plone.app.contenttypes, a behavior is provided which must be manually
+activated on the desired content types.
```

Comment:
@reinhardt shouldn't that be "on the desired content object"? Activation happens on the content object, not the type, right?